corehq/apps/app_manager/templates/app_manager/blank_form.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<h:html xmlns:h="http://www.w3.org/1999/xhtml" xmlns:orx="http://openrosa.org/jr/xforms" xmlns="http://www.w3.org/2002/xforms" xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:jr="http://openrosa.org/javarosa" xmlns:vellum="http://commcarehq.org/xforms/vellum">
+	<h:head>
+		<h:title>{{ name }}</h:title>
+		<model>
+			<instance>
+				<data xmlns:jrm="http://dev.commcarehq.org/jr/xforms" xmlns="http://openrosa.org/formdesigner/{{ xmlns }}" uiVersion="1" version="1" name="{{ name }}">
+				</data>
+			</instance>
+			<itext>
+				<translation lang="{{ lang }}" default="" />
+			</itext>
```

Comment:
I think this `<itext>` element could be omitted. Looks like vellum sets the default language based on the list of languages passed to it on initialization, so this doesn't seem to be providing any benefit.